content/source/docs/enterprise/sentinel/import/tfrun.html.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: enterprise2
+page_title: "tfrun - Imports - Sentinel - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-sentinel-imports-tfrun"
+description: |-
+    The `tfrun` import provides access to data associated with a Terraform run.
+---
+
+# Import: tfrun
+
+The `tfrun` import provides access to data associated with a [Terraform run][run-glossary].
+
+This import currently includes a single namespace - `workspace` - which provides Sentinel policies with
+static data regarding the Terraform Cloud [workspace][workspace-glossary] associated with the run.
+
+```
+tfrun
+└── workspace
+```
+
+-> Note: When policies using this import, keep in mind that workspace
+data is generally editable by users outside of the context of policy
+enforcement. For example, consider the case of omitting the enforcement of
+policy rules for development workspaces by the workspace name (allowing the
+policy to pass if the workspace ends in `-dev`). While this is useful for
+extremely granular exceptions, the workspace name could be edited by
+workspace admins, effectively bypassing the policy. In this case, where an
+extremely strict separation of policy managers vs. workspace practitioners is
+required, using [policy sets](/docs/enterprise/sentinel/manage-policies.html)
+to only enforce the policy on non-development workspaces is more appropriate.
+
+[run-glossary]: /docs/glossary.html#run
+[workspace-glossary]: /docs/glossary.html#workspace
+
+## Namespace: Workspace
+
+```
+workspace
```

Comment:
To be consistent with the rest of the imports, can we move the sub-key views into the main namespace view?